### ISSUE TITLE
PlayerQuitEvent Error

### ISSUE THREAD

**Issue by RWoof** [Ruinscraft/Panilla#61]
I just watched the Spartan plugin kick a player and Panilla brought up this error. I'm not sure what caused it. I'm running Paper 88 in Minecraft 1.16.1 and the latest versions of both Panilla 8 and Spartan. 

```
 [21:21:08] [Server thread/ERROR]: Could not pass event PlayerQuitEvent to Panilla v1.5.5
java.util.NoSuchElementException: com.ruinscraft.panilla.api.io.dplx.PacketInspectorDplx
	at io.netty.channel.DefaultChannelPipeline.getContextOrDie(DefaultChannelPipeline.java:1082) ~[patched_1.16.1.jar:git-Paper-88]
	at io.netty.channel.DefaultChannelPipeline.remove(DefaultChannelPipeline.java:417) ~[patched_1.16.1.jar:git-Paper-88]
	at com.comphenix.protocol.injector.netty.PipelineProxy.remove(PipelineProxy.java:302) ~[?:?]
	at com.ruinscraft.panilla.api.io.IPlayerInjector.unregister(IPlayerInjector.java:73) ~[?:?]
	at com.ruinscraft.panilla.bukkit.JoinQuitListener.onQuit(JoinQuitListener.java:27) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor87.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.1.jar:git-Paper-88]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.PlayerList.disconnect(PlayerList.java:509) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.PlayerConnection.a(PlayerConnection.java:1590) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.PlayerConnection.disconnect(PlayerConnection.java:291) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.craftbukkit.v1_16_R1.entity.CraftPlayer.kickPlayer(CraftPlayer.java:441) ~[patched_1.16.1.jar:git-Paper-88]
	at me.vagdedes.spartan.features.h.c.h(PunishmentExecutor.java:125) ~[?:?]
	at me.vagdedes.spartan.b.b.onCommand(Commands.java:570) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:159) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.craftbukkit.v1_16_R1.CraftServer.dispatchCommand(CraftServer.java:794) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.Bukkit.dispatchCommand(Bukkit.java:738) ~[patched_1.16.1.jar:git-Paper-88]
	at me.vagdedes.spartan.d.d.a(Config.java:475) ~[?:?]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftTask.run(CraftTask.java:99) ~[patched_1.16.1.jar:git-Paper-88]
	at org.bukkit.craftbukkit.v1_16_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:468) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.MinecraftServer.b(MinecraftServer.java:1291) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.DedicatedServer.b(DedicatedServer.java:377) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.MinecraftServer.a(MinecraftServer.java:1212) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.MinecraftServer.v(MinecraftServer.java:1000) ~[patched_1.16.1.jar:git-Paper-88]
	at net.minecraft.server.v1_16_R1.MinecraftServer.lambda$a$0(MinecraftServer.java:177) ~[patched_1.16.1.jar:git-Paper-88]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252] 
```

**Comment by dextonanderson** [comment]
It's actually an issue with Panilla. It's a harmless error, so luckily you can ignore it for now. I will have a fix out soon, though.

Spartan doesn't have an effect here.

**Comment by RWoof** [comment]
Thanks!

**Comment by dextonanderson** [comment]
Fixed in 1.5.6
